apps/webClient/src/components/View/index.tsx

```diff
@@ -7,25 +7,96 @@ import Container from "@material-ui/core/Container";
 import Box from "@material-ui/core/Box";
 import Grid from "@material-ui/core/Grid";
 import Logo from "../Logo";
+import { usePrevious } from "../../util";
 
 interface Props {
   children: React.ReactNode;
 }
 
-const RED = "rgb(177,32,41)";
 const RED_GRADIENT =
   "linear-gradient(45deg, rgba(177,32,41,1) 0%, rgba(244,121,32,1) 100%)";
-const GREEN = "rgb(14,124,63)";
 const GREEN_GRADIENT =
   "linear-gradient(45deg, rgba(14,124,63,1) 0%, rgba(190,198,64,1) 100%)";
-
-const BLUE = "rgb(28,85,119)";
 const BLUE_GRADIENT =
   "linear-gradient(45deg, rgba(28,85,119,1) 0%, rgba(46,192,209,1) 100%)";
-const YELLOW = "rgb(190,121,42)";
 const YELLOW_GRADIENT =
   "linear-gradient(45deg, rgba(190,121,42,1) 0%, rgba(254,228,104,1) 100%)";
 
+// Home Colors
+const DARK_ORANGE = "#B12029";
+const LIGHT_ORANGE = "#F47920";
+
+// About colors
+const DARK_BLUE = "#1C5577";
+const LIGHT_BLUE = "#2EC0D1";
+
+// Our Work colors
+const DARK_GREEN = "#0E7C3F";
+const LIGHT_GREEN = "#BEC640";
+
+// About colors
+const DARK_YELLOW = "#BE792A";
+const LIGHT_YELLOW = "#FEE468";
+
+const backgroundTransitionsMap = {
```

Comment:
Not sure if we still want this. Will  most likely be removed.

Reply:
```
const moreStyles = Object.keys(backgroundTransistionsMap).reduce((memo, prevPage) => {
  Object.keys(backgroundTransistionsMap[prevPage]).forEach( currentPage => {
    const data = backgroundTransistionsMap[prevPage][currentPage];
    Object.assign(memo, {
      [`${prevPage}-${currentPage}-1`]: {data[0] },
           [`${prevPage}-${currentPage}-2`]: { data[1] }
    
  })
  return memo
}, {}) 
```